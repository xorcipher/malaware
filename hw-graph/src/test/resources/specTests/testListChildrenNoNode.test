# Tests that the getChildren method works properly for a graph in which the graph does not contain the specified node.

# Return all the children of a given node within a given graph in a List.

CreateGraph graph

AddNode graph A

ListChildren graph A