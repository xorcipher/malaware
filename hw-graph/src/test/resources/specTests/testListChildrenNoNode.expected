# Tests that the getChildren method works properly for a graph in which the graph does not contain the specified node.

# Return all the children of a given node within a given graph in a List.

created graph graph

added node A to graph

the children of A in graph are: